pallets/phala/src/stakepool.rs

```diff
@@ -1218,6 +1238,7 @@ pub mod pallet {
 					pid: pool_info.pid,
 					user: user_info.user.clone(),
 					amount: reduced,
+					shares: withdrawing_shares,
```

Comment:
Be careful. `pool_info.remove_stake` may remove more or less share than `withdrawing_shares` because of the dust problem. Try to return the actual removed share in  `remove_stake()` and pass it to this event for accuracy.